pkg/admission/kubequota/kubequota_clusterworkspace_monitor.go

```diff
@@ -113,13 +113,11 @@ func (m *clusterWorkspaceDeletionMonitor) processNextWorkItem() bool {
 }
 
 func (m *clusterWorkspaceDeletionMonitor) process(key string) error {
-	_, clusterAwareName, err := cache.SplitMetaNamespaceKey(key)
+	parent, _, name, err := kcpcache.SplitMetaClusterNamespaceKey(key)
 	if err != nil {
 		runtime.HandleError(err)
 		return nil
 	}
-	// e.g. root:org<separator>ws
```

Comment:
Hm, crap - Need to do a lot more work to get comments munged correctly ... would likely need to merge this with any from the first statement, since this `SplitClusterAwareKey` node gets removed entirely